core/src/main/java/org/infinispan/configuration/cache/ClusteringConfiguration.java

```diff
@@ -19,7 +19,7 @@
          AttributeDefinition.builder("remoteTimeout", TimeUnit.SECONDS.toMillis(15)).build();
 
    static AttributeSet attributeDefinitionSet() {
-      return new AttributeSet(ClusteringConfiguration.class, CACHE_MODE);
+      return new AttributeSet(ClusteringConfiguration.class, CACHE_MODE, REMOTE_TIMEOUT);
```

Comment:
Uh, do I need to register the listener on a deprecated class? @tristantarrant please counsel...